command/version_history.go

```diff
@@ -0,0 +1,127 @@
+package command
+
+import (
+	"fmt"
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+	"github.com/ryanuber/columnize"
+	"strings"
+)
+
+var (
+	_ cli.Command             = (*VersionHistoryCommand)(nil)
+	_ cli.CommandAutocomplete = (*VersionHistoryCommand)(nil)
+)
+
+// VersionHistoryCommand is a Command implementation prints the version.
+type VersionHistoryCommand struct {
+	*BaseCommand
+}
+
+func (c *VersionHistoryCommand) Synopsis() string {
+	return "Prints the version history of the target Vault server"
+}
+
+func (c *VersionHistoryCommand) Help() string {
+	helpText := `
+Usage: vault version
```

Comment:
```suggestion
Usage: vault version-history
```

is it supposed to be version-history?

Reply:
Ah, yes! Thank you for catching that.